### DISCUSSION TITLE
option -bootclasspath not supported at compliance level 9 and above

### DISCUSSION THREAD

**Discussion by Hector-Soto** [eclipse-pde/eclipse.pde#1372] [General]
Hello, Im trying to make a very simple in eclipse to add a simple functionality to simplicity studio so i can learn how it works and develop something more complex i have in mind, the problem is that once i export it in a .jar where i have a plugin.xml, META-INF\MANIFEST.MF and com\plugin\prueba\view.class it doesnt let me export it . The plugin is in java version 17 and a compile it with java 1.8 because there was an error about the classpath i dont know why so i had to change the compiler to a version before java 9. I would like to compile with java 17 but i don´t know how to fix this error when i export the plugin: option -bootclasspath not supported at compliance level 9 and above.

**Comment by laeubi** [comment]
Please share a runnable example, as you already noticed you need java 17 but using a compile option that is no longer supported there.
